tests/test_group.py

```diff
@@ -424,11 +424,28 @@ def test_group_len(store: Store, zarr_format: ZarrFormat) -> None:
 
 def test_group_setitem(store: Store, zarr_format: ZarrFormat) -> None:
     """
-    Test the `Group.__setitem__` method.
+    Test the `Group.__setitem__` method. (Deprecated)
```

Comment:
Only comment: typically it's nice for users to say, "Deprecated: use group.create_array instead" (for example)